### PR TITLE
Adds a new drink: Peppermint Patty from Archer!

### PR DESCRIPTION
**Pull request by ghost**
:cl: Armhulen code, Keekenox sprites and S_____ as the ideas guy :roll_eyes:
add: New drinks: Peppermint Patty and the Candyland Extract!!
/:cl:

https://www.youtube.com/watch?v=njI5tHKd4v8

drink effect of Peppermint Patty: prevents coughing, heats you up
drink effect of Candyland Extract: transports you right to candyland

### CODE REVIEW

**Comment by MrDoomBringer** [comment]
S____?

**Comment by Armhulen** [comment]
I can't say that word, it'll get my pr closed.

**Comment by Armhulen** [comment]
I'll code Candyland tomorrow!

**Comment by Spaceinaba** [comment]
I don't see how mentioning shezza will get your pr closed

**Comment by Armhulen** [comment]
https://github.com/tgstation/tgstation/pull/38425

**Comment by Tlaltecuhtli** [comment]
goodbye

**Comment by Time-Green** [comment]
One crayon reagent is dominant, you wont get a cool rainbow flashing effect with that

**Comment by DaxDupont** [comment]
Does candyland extract not actually do anything?

**Comment by Armhulen** [comment]
Committing candyland after this

**Comment by Armhulen** [comment]
Whoops

**Comment by ExcessiveUseOfCobblestone** [comment]
Lego Darth Vader strikes again...

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm [hunk 1]
```diff
@@ -633,6 +633,20 @@
 	required_catalysts = list("enzyme" = 5)
 	mix_message = "The rice grains ferment into a clear, sweet-smelling liquid."
 
+/datum/chemical_reaction/peppermint_patty
+	name = "Peppermint Patty"
+	id = "peppermint_patty"
+	results = list("peppermint_patty" = 10)
+	required_reagents = list("hot_coco" = 6, "creme_de_cacao" = 1, "creme_de_menthe" = 1, "vodka" = 1, "menthol" = 1)
+	mix_message = "The coco turns mint green just as the strong scent hits your nose."
```
Comment (iskyp) [inline code/modules/food_and_drinks/recipes/drinks_recipes.dm]: cocoa not coco

**Review by ghost — CHANGES_REQUESTED** [review]
literally unplayable

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm (hunk 1 above)
Comment (Armhulen) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm]: /datum/chemical_reaction/hot_coco
--
49 | name = "Hot Coco"
50 | id = "hot_coco"
51 | results = list("hot_coco" = 5)
52 | required_reagents = list("water" = 5, "cocoa" = 1)

**Comment by Armhulen** [comment]
Hoo that took a second, but of course you can't call an atom proc on a datum :D

File: code/modules/food_and_drinks/recipes/drinks_recipes.dm (hunk 1 above)
Comment (iskyp) [reply code/modules/food_and_drinks/recipes/drinks_recipes.dm]: image

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 2]
```diff
@@ -1559,6 +1559,53 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	description = "This mesmerizing drink has it's properties from the mercury and space drugs. consumption not suggested."
+	color = "#664300"
+	taste_description = "the rainbow" //taste the rainbow!
+	boozepwr = 0
+	glass_icon_state = "candyland_extract"
+	glass_name = "Candyland Extract"
+	glass_desc = "A very nice shifting rainbow in a glass!"
+
+/datum/reagent/consumable/ethanol/candyland_extract/on_mob_add(mob/living/M)
+	for(var/turf/globalturf in world) //disaster strikes
```
Comment (Armhulen) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: so i know this aint right


but how is it wrong :)

Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Looks like you've picked up more than just S____'s ideas.

Comment (Armhulen) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: 💃 acdc - var/turf/T all around the world!

no but really why can't i get away with this

Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Have you tested it?

Comment (ExcessiveUseOfCobblestone) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: You may as well just make a new layer for turfs and add the candy overlay to them on initialize then just add and remove the ability to see them when ingested.

Comment (Armhulen) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: but wouldn't that incur the ire of maintainers

Comment (ninjanomnom) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: \>`on_mob_add`
\>`in world` loop
dear god

Comment (Armhulen) [reply code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: shoulda thought harder about it sorry

**Comment by Armhulen** [comment]
removed candyland extract, not possible codewise without it being a big mess

**Comment by ninjanomnom** [comment]
It's certainly possibly

**Pull request merged**